src/main/java/org/broadinstitute/hellbender/tools/genome/SparkGenomeReadCounts.java

```diff
@@ -0,0 +1,134 @@
+package org.broadinstitute.hellbender.tools.genome;
+
+import com.google.common.collect.Sets;
+import htsjdk.samtools.SAMSequenceDictionary;
+import htsjdk.samtools.util.Locatable;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.apache.spark.api.java.JavaRDD;
+import org.apache.spark.api.java.JavaSparkContext;
+import org.broadinstitute.hellbender.cmdline.Argument;
+import org.broadinstitute.hellbender.cmdline.CommandLineProgramProperties;
+import org.broadinstitute.hellbender.cmdline.programgroups.CopyNumberProgramGroup;
+import org.broadinstitute.hellbender.engine.filters.ReadFilter;
+import org.broadinstitute.hellbender.engine.filters.ReadFilterLibrary;
+import org.broadinstitute.hellbender.engine.filters.WellformedReadFilter;
+import org.broadinstitute.hellbender.engine.spark.GATKSparkTool;
+import org.broadinstitute.hellbender.engine.spark.datasources.ReadsSparkSource;
+import org.broadinstitute.hellbender.exceptions.UserException;
+import org.broadinstitute.hellbender.tools.exome.SampleCollection;
+import org.broadinstitute.hellbender.tools.exome.Target;
+import org.broadinstitute.hellbender.tools.exome.TargetCoverageUtils;
+import org.broadinstitute.hellbender.utils.IntervalUtils;
+import org.broadinstitute.hellbender.utils.SimpleInterval;
+import org.broadinstitute.hellbender.utils.read.GATKRead;
+
+import java.io.File;
+import java.io.PrintStream;
+import java.util.*;
+import java.util.stream.Collectors;
+
+@CommandLineProgramProperties(
+        summary = "Spark WGS Coverage",
+        oneLineSummary = "Spark WGS Coverage.",
+        programGroup = CopyNumberProgramGroup.class)
+public class SparkGenomeReadCounts extends GATKSparkTool {
+    private static final long serialVersionUID = 1l;
+
+    private static final int binsize = 10000;
```

Comment:
Add an issue to allow binsize specification through the CLI.  ``--binsize 1000``